### ISSUE TITLE
PanResponder: 'move' is called before mouse is pressed

### ISSUE THREAD

**Issue by JakeCoxon** [necolas/react-native-web#701]
I believe this is a bug

I have a carousel style component where the responder should be granted when the user presses and drags for at least 50px horizontally. It shouldn't be granted on press because it can be contained in a vertical scrollview, and also it should let buttons be pressed inside.

In react-native I can use `onMoveShouldSetPanResponder: (evt, g) => Math.abs(g.dx) > 50,`.
On mobile it's impossible to cause a move before a start due to using touches, but using a mouse pointer things are different.

I think in react native web the PanResponder shouldn't grant move until the mouse has been pressed, in order to mirror how touch events works. Otherwise doing what I want doesn't seem possible.

Glitch example:
https://glitch.com/edit/#!/bold-bit?path=src/App/index.js

As you can see the responder gets granted as soon as you move over the red box

* OS: osx
* Device: all
* Browser: chrome
* React Native for Web (version): 0.1.9
* React (version): 16

**Comment by necolas** [comment]
Yeah that shouldn't be the case. You can see the `PanResponder` [demo](https://necolas.github.io/react-native-web/storybook/?selectedKind=APIs&selectedStory=PanResponder) doesn't have this issue. Let me know if you figure out what the root cause is

**Comment by JakeCoxon** [comment]
The difference is that the PanResponder demo doesn't request grant for mouse move, only for mouse press:

```
  _handleStartShouldSetPanResponder = (e: Object, gestureState: Object): boolean => {
    // Should we become active when the user presses down on the circle?
    return true;
  };

  _handleMoveShouldSetPanResponder = (e: Object, gestureState: Object): boolean => {
    // Should we become active when the user moves a touch over the circle?
    return false;
  };
```

whereas in my code I want mouse move, but only after mouse press

**Comment by necolas** [comment]
I see. Thanks

**Comment by necolas** [comment]
I tried your demo (thanks for making one) against master. The problem looks to have been fixed by this patch - 9ee89bc7f7e0548a930d37087caa63636d490fa2. This makes the mouse work like touch in the responder system - press and drag is a move. The fix should be in the next release. Thanks again
